parser/tests/Create.test

```diff
@@ -139,6 +139,62 @@ CREATE TABLE test (attr INT) CHECK(attr>0)
                              ^
 ==
 
+# Empty partitioning attribute is not allowed
```

Comment:
Could you please add the correct queries with hash- and range- partitions first?